src/components/BorrowerProfile/DescriptionListItem.vue

```diff
@@ -1,11 +1,15 @@
 <template>
 	<div class="tw-flex tw-mb-1.5">
 		<dt class="tw-flex-1 tw-mb-0">
-			<button @click="$emit('show-definition')" class="tw-underline tw-font-book">
+			<button @click="$emit('show-definition')" class="tw-underline tw-font-book"
+				data-testid="bp-details-lightbox-btn"
```

Comment:
These would need to be used in conjunction with the parent in either LoanDetails.vue, FieldPartnerDetails.vue and TrusteeDetails.vue.

Reply:
Correct and likely some "term" matching if you want to click any specific item...

Reply:
Oh I see you have some specific wrappers below too!